src/analyze/tableau/outlier_detection/outlier_detect_ilf.py

```diff
@@ -0,0 +1,96 @@
+import sys
+if sys.version_info < (3, 6):
```

Comment:
This is a good test for Python 3.6, but I think it should be in the `main` function.  It's tempting to keep it here, because it's a convenient place to see the version requirement quickly.  However, it goes against the usual "everything in functions" practice.